src/api/.rubocop_todo.yml

```diff
@@ -200,15 +199,15 @@ Lint/UriEscapeUnescape:
     - 'spec/models/package_spec.rb'
     - 'test/functional/request_controller_test.rb'
 
-# Offense count: 915
+# Offense count: 914
 Metrics/AbcSize:
   Max: 239
 
-# Offense count: 603
+# Offense count: 68
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
-  Max: 457
+  Max: 212
```

Comment:
[Nitpick]: This Max value got changed automatically when rubocop with autocorrect was run?